server/src/main/java/org/opensearch/action/admin/cluster/decommission/awareness/put/DecommissionRequest.java

```diff
@@ -32,14 +30,19 @@ public class DecommissionRequest extends ClusterManagerNodeRequest<DecommissionR
     public static final TimeValue DEFAULT_NODE_DRAINING_TIMEOUT = TimeValue.timeValueSeconds(120);
 
     private DecommissionAttribute decommissionAttribute;
-
+    private boolean originalRequest = false;
     private TimeValue delayTimeout = DEFAULT_NODE_DRAINING_TIMEOUT;
 
     // holder for no_delay param. To avoid draining time timeout.
     private boolean noDelay = false;
 
     public DecommissionRequest() {}
 
+    public DecommissionRequest(DecommissionAttribute decommissionAttribute, boolean originalRequest) {
+        this.decommissionAttribute = decommissionAttribute;
+        this.originalRequest = originalRequest;
```

Comment:
should be removed for unintended access by users?

Reply:
yes. removing